github-app/github-checks-app-gen.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import sys
+from secrets import token_urlsafe
+from urllib.parse import urlencode, urlparse
+
+
+def init_logger():
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.INFO)
+    h = logging.StreamHandler(sys.stdout)
+    logger.addHandler(h)
+
+    return logger
+
+
+logger = init_logger()
+
+
+if __name__ == '__main__':
+    print("Please select an option cloud or enterprise server:")
+    print("1. GitHub Cloud")
+    print("2. GitHub Enterprise Server")
+    choice = input("Enter 1 or 2: ")
+
+    name = 'Hashicorp Vault Radar Test App'
+    domain = 'github.com'
+    webhook_url =  'https://api.hashicorp.cloud/2023-05-01/vault-radar/api/github-apps/events'
+    if choice == '1':
+        name = input('Enter a unique GitHub app name: ')
+    elif choice == '2':
+        domain = input('Enter your GitHub Enterprise Server domain name (i.e. github.companyname.com): ')
+        domain = urlparse(domain if '//' in domain else f'//{domain}').netloc
+        webhook_url = f'https://api.hashicorp.cloud/2023-05-01/vault-radar/api/github-apps/events?domain={domain}'
```

Comment:
nit: Append `?domain={domain}` to existing `webhook_url` ? That way if the webhook endpoint ever changes we only have to update one place?